bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -225,27 +225,28 @@ public Widget getDefaultWidget(Class<? extends Item> itemType, String itemName)
 			return SitemapFactory.eINSTANCE.createColorpicker();
 		}
 		if (itemType.equals(PlayerItem.class)) {
-		    return createSwitchWithButtons(
-		            PlayPauseType.PLAY,
-		            PlayPauseType.PAUSE,
-		            RewindFastforwardType.REWIND,
-		            RewindFastforwardType.FASTFORWARD,
-		            NextPreviousType.NEXT,
-		            NextPreviousType.PREVIOUS);
+		    return createSwitchWithButtons();
 		}
 
 		return null;
 	}
 
-    private Switch createSwitchWithButtons(Enum<?>... commands) {
+    private Switch createSwitchWithButtons() {
```

Comment:
Should this method better be named 'createPlayerItemSwitch', if it only creates the controls for a player item now?